src/core/mapping/base_mapper.cc

```diff
@@ -676,7 +675,7 @@ bool BaseMapper::map_legate_store(const Legion::Mapping::MapperContext ctx,
     regions = group->get_regions();
   }
 
-  bool created     = false;
+  bool created     = true;
```

Comment:
why the change? I would think the initial value would be irrelevant?

Reply:
This was a "drive-by" bug fix; the default value was wrong for the [case](https://github.com/nv-legate/legate.core/blob/branch-23.09/src/core/mapping/base_mapper.cc#L697) where `policy.allocation == AllocPolicy::MUST_ALLOC`, because in that case the instance always gets created. The default value doesn't matter for the [case](https://github.com/nv-legate/legate.core/blob/branch-23.09/src/core/mapping/base_mapper.cc#L684) where `policy.allocation == AllocPolicy::MAY_ALLOC`, because it will get overwritten anyway by the `find_or_create_physical_instance` call.